src/binary/requests/statusBar.ts

```diff
@@ -0,0 +1,30 @@
+
+import { tabNineProcess } from "./requests";
+
+export enum StatusBarActionActions {
+  NONE = 0,
+  OPEN_LP = 1,
```

Comment:
I did a mistake when I did this thing in the notifications, and we don't need to repeat it. The binary is opening the LP so this air conditioning placement is redundant and will bite us in the future.

Reply:
In the minimum, just omit the OPEN_LP option so it will be clear that nothing is done